### PR TITLE
Update tooltips and other helpers

### PR DESCRIPTION
**Pull request by ciduarte**
DISCO-1109

### CODE REVIEW

**Comment by ciduarte** [comment]
For reference, "extra text" was reformatted to appear under the label like this:

Screen Shot 2019-07-08 at 1 43 54 PM

File: src/components/EditCoursePage/EditCourseForm.jsx [hunk 1]
```diff
@@ -307,6 +308,7 @@ export class BaseEditCourseForm extends React.Component {
                       </p>
                     </div>
                   }
+                  extraText="Image must be 1134x675 pixels in size."
```
Comment (mikix) [inline src/components/EditCoursePage/EditCourseForm.jsx]: This does feel like we can pull this out of the tooltip now? Or did Carol want it in both places?

File: src/components/FieldLabel/index.jsx [hunk 2]
```diff
@@ -21,7 +21,7 @@ const FieldLabel = ({
       <strong>{text}</strong>
       {helpText && id && <FieldHelp id={`${id}-help`} tip={helpText} />}
       {requireText}
-      {extraText}
+      <div><span className="text-muted">{extraText}</span></div>
```
Comment (mikix) [inline src/components/FieldLabel/index.jsx]: Eh, you *could* put the className on the div, not the span. But it's all the same.

File: src/components/EditCoursePage/EditCourseForm.jsx (hunk 1 above)
Comment (ciduarte) [reply src/components/EditCoursePage/EditCourseForm.jsx]: Good question! Checked with Carol and she said both places

**Pull request merged**